packages/web/src/alliance/PinWheel.tsx

```diff
@@ -0,0 +1,213 @@
+                    d={path}
+                    style={[styles.base, styles.cycle, baseCoinStyle, style]}
+                  />
+                )
+              })}
+            </G>
+          )}
+          <Quadrants animate={!this.props.static} />
+        </Svg>
+        {this.props.children && (
+          <View style={isMobile ? standardStyles.centered : styles.absoluteCenter}>
+            {this.props.children}
+          </View>
+        )}
+      </View>
+    )
+  }
+}
+
+export default withScreenSize(PinWheel)
+
+interface QuadrantProps {
+  animate: boolean
+}
+class Quadrants extends React.PureComponent<QuadrantProps> {
+  render() {
+    return (
+      <G>
+        {VECTORS.map((path, index) => {
+          return (
+            <Path
+              key={path.slice(0, 11)}
+              d={path}
+              style={[styles.justCoin, this.props.animate ? animated(index) : still(index)]}
+            />
+          )
+        })}
+      </G>
+    )
+  }
+}
+
+const DELAYMENT = 30
+const QUARTERIZER = 200
+
+function animated(index: number) {
+  return {
+    mixBlendMode: 'screen',
+    animationIterationCount: 1,
+    animationDelay: `${INTERVAL_MS + index * DELAYMENT}ms`,
+    animationFillMode: 'both',
+    animationDuration: `${DURATION / 3}ms`,
+    animationKeyframes: solidFadeInFrames(
+      quartColor(index),
+      ((index * index) / VECTORS.length) * QUARTERIZER
+    ),
+  }
+}
+
+function still(index: number) {
+  return {
+    mixBlendMode: 'screen',
+    ...colorFrame(quartColor(index)),
+  }
+}
+
+function quartColor(index: number) {
+  const quarter = VECTORS.length / 4
+  if (index < quarter) {
+    return colors.greenScreen
+  } else if (index < quarter * 2) {
+    return colors.purpleScreen
+  } else if (index < quarter * 3) {
+    return colors.blueScreen
+  } else {
+    return colors.redScreen
+  }
+}
+
+const styles = StyleSheet.create({
+  justCoin: {
+    opacity: 0.3,
+    fill: 'transparent',
+  },
+  base: {
+    opacity: 0.2,
+    animationIterationCount: 3,
+    animationDuration: `${DURATION}ms`,
+    animationFillMode: 'both',
+    animationDirection: 'normal',
+  },
+  cycle: {
+    animationKeyframes: getKeyframes(),
+  },
+  sweepContainer: {
+    width: '100%',
+    height: '100vh',
+    maxHeight: '100vw',
+    maxWidth: '90vw',
+  },
+  lightingOff: {
+    opacity: 0,
+  },
+  lighting: {
+    height: '100%',
+    width: '100%',
+    animationIterationCount: 1,
+    animationDuration: `${DURATION * 2}ms`,
+    animationDelay: `${DURATION}ms`,
+    animationFillMode: 'both',
+    animationKeyframes: [
+      {
+        '0%': {
+          opacity: 0,
+        },
+        '60%': {
+          opacity: 0.9,
+        },
+        '100%': {
+          opacity: 0,
+        },
+      },
+    ],
+  },
+  absoluteCenter: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    left: 0,
+    bottom: 0,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+})
+
+function getKeyframes() {
```

Comment:
Nit why is this wrapped in a getter if we don't have params for it? Should this be an array at module scope?